.tekton/openshift-builds-operator-pull-request.yaml

```diff
@@ -40,8 +40,12 @@ spec:
     value: "true"
   - name: hermetic
     value: "true"
+  - name: build-platforms
+    value:
+      - linux/x86_64
+      - linux/arm64
   - name: prefetch-input
-    value: '{"packages": [{"type": "gomod"}], "flags": ["gomod-vendor-check"]}'
+    value: '{"packages": [{"type": "gomod"}]}'
```

Comment:
Why are we dropping the `gomod-vendor-check` flag?

Reply:
This flag is deprecated because and in the latest iteration, Cachi2 automatically checks the vendor.

From logs..

```
STEP-PREFETCH-DEPENDENCIES

INFO: Using mounted CA bundle: /mnt/trusted-ca/ca-bundle.crt
'/mnt/trusted-ca/ca-bundle.crt' -> '/etc/pki/ca-trust/source/anchors/ca-bundle.crt'
2025-01-12 02:42:43,966 WARNING The `gomod-vendor-check` flag is deprecated and will be removed in future versions. Its use is no longer necessary, Cachi2 will automatically check the contents of the vendor directory in case it is present.
2025-01-12 02:42:45,230 INFO Fetching the gomod dependencies at subpath .
2025-01-12 02:42:45,230 INFO Fetching the gomod dependencies at the "." directory
2025-01-12 02:42:50,137 INFO go.mod reported versions: '1.22.0'[go], '1.22.7'[toolchain]
```